data/mods/gen1/random-teams.ts

```diff
@@ -234,6 +230,29 @@ export class RandomGen1Teams extends RandomGen2Teams {
 		return pokemon;
 	}
 
+	shouldCullMove(move: Move, hasMove: {[k: string]: true}, counter: {[k: string]: any}): {cull: boolean} {
+		switch (move.id) {
+		// bit redundant to have both, but neither particularly better than the other
+		case 'hydropump':
+			return {cull: hasMove['surf']};
+		case 'surf':
+			return {cull: hasMove['hydropump']};
+
+		// other redundancies that aren't handled within the movesets themselves
+		case 'selfdestruct':
+			return {cull: hasMove['rest']};
+		case 'rest':
+			return {cull: hasMove['selfdestruct']};
+		case 'sharpen': case 'swordsdance':
+			return {cull: counter.Special > counter.Physical || !counter.Physical || hasMove['growth']};
+		case 'growth':
+			return {cull: counter.Special < counter.Physical || !counter['Special'] || hasMove['swordsdance']};
```

Comment:
why does this use `counter['Special']` and not `counter.Special`?

Reply:
My find and replace regex missed it for some reason.